src/ldc/attributes.d

```diff
@@ -118,6 +118,28 @@ private struct _dynamicCompileConst
 {
 }
 
+/++
+ + When applied to a function, marks this function for dynamic compilation.
+ + Static version of function will be called via direct call but function
+ + body will still be available for jit facilities (e.g. jit bind).
```

Comment:
I'd probably prefer `@dynamicCompilable`, I find it simpler to grasp ('can be compiled at runtime too').